packages/core/src/components/Icon/__stories__/Icon.stories.tsx

```diff
@@ -31,13 +31,13 @@ export const Overview = {
 };
 
 export const VibeIcon = {
-  render: () => <Icon iconType={Icon.type.SVG} icon={Bolt} iconLabel="my bolt svg icon" iconSize={16} />,
```

Comment:
aria-label shouldn't be added for non-interactive elements, right? I wonder why has it been added here in the first place